<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="Integration test">
<testcase name="compiler_tests/_example/example.c">
</testcase>
<testcase name="compiler_tests/array/declare_global.c">
<error type="error" message="Fail: see ./bin/output/compiler_tests_array_declare_global.linker.stderr.log and ./bin/output/compiler_tests_array_declare_global.linker.stdout.log">Fail: see ./bin/output/compiler_tests_array_declare_global.linker.stderr.log and ./bin/output/compiler_tests_array_declare_global.linker.stdout.log</error>
</testcase>
<testcase name="compiler_tests/array/declare_local.c">
<error type="error" message="Fail: simulation did not exit with exit-code 0">Fail: simulation did not exit with exit-code 0</error>
</testcase>
<testcase name="compiler_tests/array/index_constant.c">
<error type="error" message="Fail: simulation did not exit with exit-code 0">Fail: simulation did not exit with exit-code 0</error>
</testcase>
<testcase name="compiler_tests/array/index_expression.c">
<error type="error" message="Fail: see ./bin/output/compiler_tests_array_index_expression.compiler.stderr.log and ./bin/output/compiler_tests_array_index_expression.compiler.stdout.log">Fail: see ./bin/output/compiler_tests_array_index_expression.compiler.stderr.log and ./bin/output/compiler_tests_array_index_expression.compiler.stdout.log</error>
</testcase>
<testcase name="compiler_tests/array/index_variable.c">
<error type="error" message="Fail: see ./bin/output/compiler_tests_array_index_variable.compiler.stderr.log and ./bin/output/compiler_tests_array_index_variable.compiler.stdout.log">Fail: see ./bin/output/compiler_tests_array_index_variable.compiler.stderr.log and ./bin/output/compiler_tests_array_index_variable.compiler.stdout.log</error>
</testcase>
<testcase name="compiler_tests/control_flow/for_multiple.c">
<error type="error" message="Fail: see ./bin/output/compiler_tests_control_flow_for_multiple.compiler.stderr.log and ./bin/output/compiler_tests_control_flow_for_multiple.compiler.stdout.log">Fail: see ./bin/output/compiler_tests_control_flow_for_multiple.compiler.stderr.log and ./bin/output/compiler_tests_control_flow_for_multiple.compiler.stdout.log</error>
</testcase>
<testcase name="compiler_tests/control_flow/for_one.c">
<error type="error" message="Fail: simulation did not exit with exit-code 0">Fail: simulation did not exit with exit-code 0</error>
</testcase>
<testcase name="compiler_tests/control_flow/for_zero_v1.c">
<error type="error" message="Fail: simulation did not exit with exit-code 0">Fail: simulation did not exit with exit-code 0</error>
</testcase>
<testcase name="compiler_tests/control_flow/for_zero_v2.c">
<error type="error" message="Fail: simulation did not exit with exit-code 0">Fail: simulation did not exit with exit-code 0</error>
</testcase>
<testcase name="compiler_tests/control_flow/if_else_false.c">
<error type="error" message="Fail: simulation did not exit with exit-code 0">Fail: simulation did not exit with exit-code 0</error>
</testcase>
<testcase name="compiler_tests/control_flow/if_else_true.c">
<error type="error" message="Fail: simulation did not exit with exit-code 0">Fail: simulation did not exit with exit-code 0</error>
</testcase>
<testcase name="compiler_tests/control_flow/if_false.c">
<error type="error" message="Fail: simulation did not exit with exit-code 0">Fail: simulation did not exit with exit-code 0</error>
</testcase>
<testcase name="compiler_tests/control_flow/if_true.c">
<error type="error" message="Fail: simulation did not exit with exit-code 0">Fail: simulation did not exit with exit-code 0</error>
</testcase>
<testcase name="compiler_tests/control_flow/sequence.c">
<error type="error" message="Fail: simulation did not exit with exit-code 0">Fail: simulation did not exit with exit-code 0</error>
</testcase>
<testcase name="compiler_tests/control_flow/while_multiple.c">
<error type="error" message="Fail: see ./bin/output/compiler_tests_control_flow_while_multiple.compiler.stderr.log and ./bin/output/compiler_tests_control_flow_while_multiple.compiler.stdout.log">Fail: see ./bin/output/compiler_tests_control_flow_while_multiple.compiler.stderr.log and ./bin/output/compiler_tests_control_flow_while_multiple.compiler.stdout.log</error>
</testcase>
<testcase name="compiler_tests/control_flow/while_once.c">
<error type="error" message="Fail: see ./bin/output/compiler_tests_control_flow_while_once.compiler.stderr.log and ./bin/output/compiler_tests_control_flow_while_once.compiler.stdout.log">Fail: see ./bin/output/compiler_tests_control_flow_while_once.compiler.stderr.log and ./bin/output/compiler_tests_control_flow_while_once.compiler.stdout.log</error>
</testcase>
<testcase name="compiler_tests/control_flow/while_zero.c">
<error type="error" message="Fail: simulation did not exit with exit-code 0">Fail: simulation did not exit with exit-code 0</error>
</testcase>
<testcase name="compiler_tests/default/test_ADD0.c">
<error type="error" message="Fail: see ./bin/output/compiler_tests_default_test_ADD0.linker.stderr.log and ./bin/output/compiler_tests_default_test_ADD0.linker.stdout.log">Fail: see ./bin/output/compiler_tests_default_test_ADD0.linker.stderr.log and ./bin/output/compiler_tests_default_test_ADD0.linker.stdout.log</error>
</testcase>
<testcase name="compiler_tests/default/test_ADD1.c">
<error type="error" message="Fail: see ./bin/output/compiler_tests_default_test_ADD1.linker.stderr.log and ./bin/output/compiler_tests_default_test_ADD1.linker.stdout.log">Fail: see ./bin/output/compiler_tests_default_test_ADD1.linker.stderr.log and ./bin/output/compiler_tests_default_test_ADD1.linker.stdout.log</error>
</testcase>
<testcase name="compiler_tests/default/test_CALL.c">
<error type="error" message="Fail: see ./bin/output/compiler_tests_default_test_CALL.linker.stderr.log and ./bin/output/compiler_tests_default_test_CALL.linker.stdout.log">Fail: see ./bin/output/compiler_tests_default_test_CALL.linker.stderr.log and ./bin/output/compiler_tests_default_test_CALL.linker.stdout.log</error>
</testcase>
<testcase name="compiler_tests/default/test_LOCAL.c">
<error type="error" message="Fail: simulation did not exit with exit-code 0">Fail: simulation did not exit with exit-code 0</error>
</testcase>
<testcase name="compiler_tests/default/test_RETURN.c">
</testcase>
<testcase name="compiler_tests/float/add_double.c">
<error type="error" message="Fail: see ./bin/output/compiler_tests_float_add_double.linker.stderr.log and ./bin/output/compiler_tests_float_add_double.linker.stdout.log">Fail: see ./bin/output/compiler_tests_float_add_double.linker.stderr.log and ./bin/output/compiler_tests_float_add_double.linker.stdout.log</error>
</testcase>
<testcase name="compiler_tests/float/add.c">
<error type="error" message="Fail: see ./bin/output/compiler_tests_float_add.linker.stderr.log and ./bin/output/compiler_tests_float_add.linker.stdout.log">Fail: see ./bin/output/compiler_tests_float_add.linker.stderr.log and ./bin/output/compiler_tests_float_add.linker.stdout.log</error>
</testcase>
<testcase name="compiler_tests/float/add_mul.c">
<error type="error" message="Fail: see ./bin/output/compiler_tests_float_add_mul.linker.stderr.log and ./bin/output/compiler_tests_float_add_mul.linker.stdout.log">Fail: see ./bin/output/compiler_tests_float_add_mul.linker.stderr.log and ./bin/output/compiler_tests_float_add_mul.linker.stdout.log</error>
</testcase>
<testcase name="compiler_tests/float/mul_add.c">
<error type="error" message="Fail: see ./bin/output/compiler_tests_float_mul_add.linker.stderr.log and ./bin/output/compiler_tests_float_mul_add.linker.stdout.log">Fail: see ./bin/output/compiler_tests_float_mul_add.linker.stderr.log and ./bin/output/compiler_tests_float_mul_add.linker.stdout.log</error>
</testcase>
<testcase name="compiler_tests/float/mul_double.c">
<error type="error" message="Fail: see ./bin/output/compiler_tests_float_mul_double.linker.stderr.log and ./bin/output/compiler_tests_float_mul_double.linker.stdout.log">Fail: see ./bin/output/compiler_tests_float_mul_double.linker.stderr.log and ./bin/output/compiler_tests_float_mul_double.linker.stdout.log</error>
</testcase>
<testcase name="compiler_tests/float/mul.c">
<error type="error" message="Fail: see ./bin/output/compiler_tests_float_mul.linker.stderr.log and ./bin/output/compiler_tests_float_mul.linker.stdout.log">Fail: see ./bin/output/compiler_tests_float_mul.linker.stderr.log and ./bin/output/compiler_tests_float_mul.linker.stdout.log</error>
</testcase>
<testcase name="compiler_tests/float/pow.c">
<error type="error" message="Fail: see ./bin/output/compiler_tests_float_pow.compiler.stderr.log and ./bin/output/compiler_tests_float_pow.compiler.stdout.log">Fail: see ./bin/output/compiler_tests_float_pow.compiler.stderr.log and ./bin/output/compiler_tests_float_pow.compiler.stdout.log</error>
</testcase>
<testcase name="compiler_tests/functions/call_constant_external.c">
<error type="error" message="Fail: see ./bin/output/compiler_tests_functions_call_constant_external.linker.stderr.log and ./bin/output/compiler_tests_functions_call_constant_external.linker.stdout.log">Fail: see ./bin/output/compiler_tests_functions_call_constant_external.linker.stderr.log and ./bin/output/compiler_tests_functions_call_constant_external.linker.stdout.log</error>
</testcase>
<testcase name="compiler_tests/functions/call_constant_internal.c">
<error type="error" message="Fail: see ./bin/output/compiler_tests_functions_call_constant_internal.linker.stderr.log and ./bin/output/compiler_tests_functions_call_constant_internal.linker.stdout.log">Fail: see ./bin/output/compiler_tests_functions_call_constant_internal.linker.stderr.log and ./bin/output/compiler_tests_functions_call_constant_internal.linker.stdout.log</error>
</testcase>
<testcase name="compiler_tests/functions/call_five_args_external.c">
<error type="error" message="Fail: see ./bin/output/compiler_tests_functions_call_five_args_external.linker.stderr.log and ./bin/output/compiler_tests_functions_call_five_args_external.linker.stdout.log">Fail: see ./bin/output/compiler_tests_functions_call_five_args_external.linker.stderr.log and ./bin/output/compiler_tests_functions_call_five_args_external.linker.stdout.log</error>
</testcase>
<testcase name="compiler_tests/functions/call_five_args_internal.c">
<error type="error" message="Fail: see ./bin/output/compiler_tests_functions_call_five_args_internal.linker.stderr.log and ./bin/output/compiler_tests_functions_call_five_args_internal.linker.stdout.log">Fail: see ./bin/output/compiler_tests_functions_call_five_args_internal.linker.stderr.log and ./bin/output/compiler_tests_functions_call_five_args_internal.linker.stdout.log</error>
</testcase>
<testcase name="compiler_tests/functions/call_identity_external.c">
<error type="error" message="Fail: see ./bin/output/compiler_tests_functions_call_identity_external.linker.stderr.log and ./bin/output/compiler_tests_functions_call_identity_external.linker.stdout.log">Fail: see ./bin/output/compiler_tests_functions_call_identity_external.linker.stderr.log and ./bin/output/compiler_tests_functions_call_identity_external.linker.stdout.log</error>
</testcase>
<testcase name="compiler_tests/functions/call_identity_internal.c">
<error type="error" message="Fail: see ./bin/output/compiler_tests_functions_call_identity_internal.linker.stderr.log and ./bin/output/compiler_tests_functions_call_identity_internal.linker.stdout.log">Fail: see ./bin/output/compiler_tests_functions_call_identity_internal.linker.stderr.log and ./bin/output/compiler_tests_functions_call_identity_internal.linker.stdout.log</error>
</testcase>
<testcase name="compiler_tests/functions/call_mutual_recursive.c">
<error type="error" message="Fail: see ./bin/output/compiler_tests_functions_call_mutual_recursive.linker.stderr.log and ./bin/output/compiler_tests_functions_call_mutual_recursive.linker.stdout.log">Fail: see ./bin/output/compiler_tests_functions_call_mutual_recursive.linker.stderr.log and ./bin/output/compiler_tests_functions_call_mutual_recursive.linker.stdout.log</error>
</testcase>
<testcase name="compiler_tests/functions/call_recursive_internal.c">
<error type="error" message="Fail: see ./bin/output/compiler_tests_functions_call_recursive_internal.linker.stderr.log and ./bin/output/compiler_tests_functions_call_recursive_internal.linker.stdout.log">Fail: see ./bin/output/compiler_tests_functions_call_recursive_internal.linker.stderr.log and ./bin/output/compiler_tests_functions_call_recursive_internal.linker.stdout.log</error>
</testcase>
<testcase name="compiler_tests/functions/call_two_args_external.c">
<error type="error" message="Fail: see ./bin/output/compiler_tests_functions_call_two_args_external.linker.stderr.log and ./bin/output/compiler_tests_functions_call_two_args_external.linker.stdout.log">Fail: see ./bin/output/compiler_tests_functions_call_two_args_external.linker.stderr.log and ./bin/output/compiler_tests_functions_call_two_args_external.linker.stdout.log</error>
</testcase>
<testcase name="compiler_tests/functions/call_two_args_internal.c">
<error type="error" message="Fail: see ./bin/output/compiler_tests_functions_call_two_args_internal.linker.stderr.log and ./bin/output/compiler_tests_functions_call_two_args_internal.linker.stdout.log">Fail: see ./bin/output/compiler_tests_functions_call_two_args_internal.linker.stderr.log and ./bin/output/compiler_tests_functions_call_two_args_internal.linker.stdout.log</error>
</testcase>
<testcase name="compiler_tests/integer/add.c">
<error type="error" message="Fail: see ./bin/output/compiler_tests_integer_add.linker.stderr.log and ./bin/output/compiler_tests_integer_add.linker.stdout.log">Fail: see ./bin/output/compiler_tests_integer_add.linker.stderr.log and ./bin/output/compiler_tests_integer_add.linker.stdout.log</error>
</testcase>
<testcase name="compiler_tests/integer/bitwise_and.c">
<error type="error" message="Fail: see ./bin/output/compiler_tests_integer_bitwise_and.linker.stderr.log and ./bin/output/compiler_tests_integer_bitwise_and.linker.stdout.log">Fail: see ./bin/output/compiler_tests_integer_bitwise_and.linker.stderr.log and ./bin/output/compiler_tests_integer_bitwise_and.linker.stdout.log</error>
</testcase>
<testcase name="compiler_tests/integer/bitwise_or.c">
<error type="error" message="Fail: see ./bin/output/compiler_tests_integer_bitwise_or.linker.stderr.log and ./bin/output/compiler_tests_integer_bitwise_or.linker.stdout.log">Fail: see ./bin/output/compiler_tests_integer_bitwise_or.linker.stderr.log and ./bin/output/compiler_tests_integer_bitwise_or.linker.stdout.log</error>
</testcase>
<testcase name="compiler_tests/integer/bitwise_xor.c">
<error type="error" message="Fail: see ./bin/output/compiler_tests_integer_bitwise_xor.linker.stderr.log and ./bin/output/compiler_tests_integer_bitwise_xor.linker.stdout.log">Fail: see ./bin/output/compiler_tests_integer_bitwise_xor.linker.stderr.log and ./bin/output/compiler_tests_integer_bitwise_xor.linker.stdout.log</error>
</testcase>
<testcase name="compiler_tests/integer/div.c">
<error type="error" message="Fail: see ./bin/output/compiler_tests_integer_div.linker.stderr.log and ./bin/output/compiler_tests_integer_div.linker.stdout.log">Fail: see ./bin/output/compiler_tests_integer_div.linker.stderr.log and ./bin/output/compiler_tests_integer_div.linker.stdout.log</error>
</testcase>
<testcase name="compiler_tests/integer/equal.c">
<error type="error" message="Fail: see ./bin/output/compiler_tests_integer_equal.linker.stderr.log and ./bin/output/compiler_tests_integer_equal.linker.stdout.log">Fail: see ./bin/output/compiler_tests_integer_equal.linker.stderr.log and ./bin/output/compiler_tests_integer_equal.linker.stdout.log</error>
</testcase>
<testcase name="compiler_tests/integer/less_than.c">
<error type="error" message="Fail: see ./bin/output/compiler_tests_integer_less_than.linker.stderr.log and ./bin/output/compiler_tests_integer_less_than.linker.stdout.log">Fail: see ./bin/output/compiler_tests_integer_less_than.linker.stderr.log and ./bin/output/compiler_tests_integer_less_than.linker.stdout.log</error>
</testcase>
<testcase name="compiler_tests/integer/less_than_equal.c">
<error type="error" message="Fail: see ./bin/output/compiler_tests_integer_less_than_equal.linker.stderr.log and ./bin/output/compiler_tests_integer_less_than_equal.linker.stdout.log">Fail: see ./bin/output/compiler_tests_integer_less_than_equal.linker.stderr.log and ./bin/output/compiler_tests_integer_less_than_equal.linker.stdout.log</error>
</testcase>
<testcase name="compiler_tests/integer/logical_and.c">
<error type="error" message="Fail: see ./bin/output/compiler_tests_integer_logical_and.linker.stderr.log and ./bin/output/compiler_tests_integer_logical_and.linker.stdout.log">Fail: see ./bin/output/compiler_tests_integer_logical_and.linker.stderr.log and ./bin/output/compiler_tests_integer_logical_and.linker.stdout.log</error>
</testcase>
<testcase name="compiler_tests/integer/logical_or.c">
<error type="error" message="Fail: see ./bin/output/compiler_tests_integer_logical_or.linker.stderr.log and ./bin/output/compiler_tests_integer_logical_or.linker.stdout.log">Fail: see ./bin/output/compiler_tests_integer_logical_or.linker.stderr.log and ./bin/output/compiler_tests_integer_logical_or.linker.stdout.log</error>
</testcase>
<testcase name="compiler_tests/integer/mul.c">
<error type="error" message="Fail: see ./bin/output/compiler_tests_integer_mul.linker.stderr.log and ./bin/output/compiler_tests_integer_mul.linker.stdout.log">Fail: see ./bin/output/compiler_tests_integer_mul.linker.stderr.log and ./bin/output/compiler_tests_integer_mul.linker.stdout.log</error>
</testcase>
<testcase name="compiler_tests/integer/sub.c">
<error type="error" message="Fail: see ./bin/output/compiler_tests_integer_sub.linker.stderr.log and ./bin/output/compiler_tests_integer_sub.linker.stdout.log">Fail: see ./bin/output/compiler_tests_integer_sub.linker.stderr.log and ./bin/output/compiler_tests_integer_sub.linker.stdout.log</error>
</testcase>
<testcase name="compiler_tests/local_var/constant_initialiser.c">
<error type="error" message="Fail: simulation did not exit with exit-code 0">Fail: simulation did not exit with exit-code 0</error>
</testcase>
<testcase name="compiler_tests/local_var/dual_var.c">
<error type="error" message="Fail: simulation did not exit with exit-code 0">Fail: simulation did not exit with exit-code 0</error>
</testcase>
<testcase name="compiler_tests/local_var/expression_initialiser.c">
<error type="error" message="Fail: simulation did not exit with exit-code 0">Fail: simulation did not exit with exit-code 0</error>
</testcase>
<testcase name="compiler_tests/local_var/identity.c">
<error type="error" message="Fail: see ./bin/output/compiler_tests_local_var_identity.linker.stderr.log and ./bin/output/compiler_tests_local_var_identity.linker.stdout.log">Fail: see ./bin/output/compiler_tests_local_var_identity.linker.stderr.log and ./bin/output/compiler_tests_local_var_identity.linker.stdout.log</error>
</testcase>
<testcase name="compiler_tests/local_var/return_constant.c">
</testcase>
<testcase name="compiler_tests/local_var/scoped_var.c">
<error type="error" message="Fail: simulation did not exit with exit-code 0">Fail: simulation did not exit with exit-code 0</error>
</testcase>
<testcase name="compiler_tests/local_var/single_var.c">
<error type="error" message="Fail: simulation did not exit with exit-code 0">Fail: simulation did not exit with exit-code 0</error>
</testcase>
<testcase name="compiler_tests/misc/enum1.c">
<error type="error" message="Fail: see ./bin/output/compiler_tests_misc_enum1.compiler.stderr.log and ./bin/output/compiler_tests_misc_enum1.compiler.stdout.log">Fail: see ./bin/output/compiler_tests_misc_enum1.compiler.stderr.log and ./bin/output/compiler_tests_misc_enum1.compiler.stdout.log</error>
</testcase>
<testcase name="compiler_tests/misc/enum2.c">
<error type="error" message="Fail: see ./bin/output/compiler_tests_misc_enum2.compiler.stderr.log and ./bin/output/compiler_tests_misc_enum2.compiler.stdout.log">Fail: see ./bin/output/compiler_tests_misc_enum2.compiler.stderr.log and ./bin/output/compiler_tests_misc_enum2.compiler.stdout.log</error>
</testcase>
<testcase name="compiler_tests/misc/switch1.c">
<error type="error" message="Fail: see ./bin/output/compiler_tests_misc_switch1.linker.stderr.log and ./bin/output/compiler_tests_misc_switch1.linker.stdout.log">Fail: see ./bin/output/compiler_tests_misc_switch1.linker.stderr.log and ./bin/output/compiler_tests_misc_switch1.linker.stdout.log</error>
</testcase>
<testcase name="compiler_tests/misc/switch2.c">
<error type="error" message="Fail: see ./bin/output/compiler_tests_misc_switch2.compiler.stderr.log and ./bin/output/compiler_tests_misc_switch2.compiler.stdout.log">Fail: see ./bin/output/compiler_tests_misc_switch2.compiler.stderr.log and ./bin/output/compiler_tests_misc_switch2.compiler.stdout.log</error>
</testcase>
<testcase name="compiler_tests/misc/typedef1.c">
<error type="error" message="Fail: see ./bin/output/compiler_tests_misc_typedef1.compiler.stderr.log and ./bin/output/compiler_tests_misc_typedef1.compiler.stdout.log">Fail: see ./bin/output/compiler_tests_misc_typedef1.compiler.stderr.log and ./bin/output/compiler_tests_misc_typedef1.compiler.stdout.log</error>
</testcase>
<testcase name="compiler_tests/misc/typedef2.c">
<error type="error" message="Fail: see ./bin/output/compiler_tests_misc_typedef2.compiler.stderr.log and ./bin/output/compiler_tests_misc_typedef2.compiler.stdout.log">Fail: see ./bin/output/compiler_tests_misc_typedef2.compiler.stderr.log and ./bin/output/compiler_tests_misc_typedef2.compiler.stdout.log</error>
</testcase>
<testcase name="compiler_tests/pointer/addressof.c">
<error type="error" message="Fail: see ./bin/output/compiler_tests_pointer_addressof.compiler.stderr.log and ./bin/output/compiler_tests_pointer_addressof.compiler.stdout.log">Fail: see ./bin/output/compiler_tests_pointer_addressof.compiler.stderr.log and ./bin/output/compiler_tests_pointer_addressof.compiler.stdout.log</error>
</testcase>
<testcase name="compiler_tests/pointer/assign.c">
<error type="error" message="Fail: see ./bin/output/compiler_tests_pointer_assign.compiler.stderr.log and ./bin/output/compiler_tests_pointer_assign.compiler.stdout.log">Fail: see ./bin/output/compiler_tests_pointer_assign.compiler.stderr.log and ./bin/output/compiler_tests_pointer_assign.compiler.stdout.log</error>
</testcase>
<testcase name="compiler_tests/pointer/dereference.c">
<error type="error" message="Fail: see ./bin/output/compiler_tests_pointer_dereference.compiler.stderr.log and ./bin/output/compiler_tests_pointer_dereference.compiler.stdout.log">Fail: see ./bin/output/compiler_tests_pointer_dereference.compiler.stderr.log and ./bin/output/compiler_tests_pointer_dereference.compiler.stdout.log</error>
</testcase>
<testcase name="compiler_tests/programs/fibonacci.c">
<error type="error" message="Fail: see ./bin/output/compiler_tests_programs_fibonacci.compiler.stderr.log and ./bin/output/compiler_tests_programs_fibonacci.compiler.stdout.log">Fail: see ./bin/output/compiler_tests_programs_fibonacci.compiler.stderr.log and ./bin/output/compiler_tests_programs_fibonacci.compiler.stdout.log</error>
</testcase>
<testcase name="compiler_tests/programs/multiply.c">
<error type="error" message="Fail: see ./bin/output/compiler_tests_programs_multiply.compiler.stderr.log and ./bin/output/compiler_tests_programs_multiply.compiler.stdout.log">Fail: see ./bin/output/compiler_tests_programs_multiply.compiler.stderr.log and ./bin/output/compiler_tests_programs_multiply.compiler.stdout.log</error>
</testcase>
<testcase name="compiler_tests/programs/sqrt.c">
<error type="error" message="Fail: see ./bin/output/compiler_tests_programs_sqrt.compiler.stderr.log and ./bin/output/compiler_tests_programs_sqrt.compiler.stdout.log">Fail: see ./bin/output/compiler_tests_programs_sqrt.compiler.stderr.log and ./bin/output/compiler_tests_programs_sqrt.compiler.stdout.log</error>
</testcase>
<testcase name="compiler_tests/strings/escaped.c">
<error type="error" message="Fail: see ./bin/output/compiler_tests_strings_escaped.compiler.stderr.log and ./bin/output/compiler_tests_strings_escaped.compiler.stdout.log">Fail: see ./bin/output/compiler_tests_strings_escaped.compiler.stderr.log and ./bin/output/compiler_tests_strings_escaped.compiler.stdout.log</error>
</testcase>
<testcase name="compiler_tests/strings/literal.c">
<error type="error" message="Fail: see ./bin/output/compiler_tests_strings_literal.compiler.stderr.log and ./bin/output/compiler_tests_strings_literal.compiler.stdout.log">Fail: see ./bin/output/compiler_tests_strings_literal.compiler.stderr.log and ./bin/output/compiler_tests_strings_literal.compiler.stdout.log</error>
</testcase>
<testcase name="compiler_tests/strings/puts.c">
<error type="error" message="Fail: see ./bin/output/compiler_tests_strings_puts.compiler.stderr.log and ./bin/output/compiler_tests_strings_puts.compiler.stdout.log">Fail: see ./bin/output/compiler_tests_strings_puts.compiler.stderr.log and ./bin/output/compiler_tests_strings_puts.compiler.stdout.log</error>
</testcase>
<testcase name="compiler_tests/strings/search.c">
<error type="error" message="Fail: see ./bin/output/compiler_tests_strings_search.compiler.stderr.log and ./bin/output/compiler_tests_strings_search.compiler.stdout.log">Fail: see ./bin/output/compiler_tests_strings_search.compiler.stderr.log and ./bin/output/compiler_tests_strings_search.compiler.stdout.log</error>
</testcase>
<testcase name="compiler_tests/struct/sizeof.c">
<error type="error" message="Fail: see ./bin/output/compiler_tests_struct_sizeof.compiler.stderr.log and ./bin/output/compiler_tests_struct_sizeof.compiler.stdout.log">Fail: see ./bin/output/compiler_tests_struct_sizeof.compiler.stderr.log and ./bin/output/compiler_tests_struct_sizeof.compiler.stdout.log</error>
</testcase>
<testcase name="compiler_tests/struct/struct_inst.c">
<error type="error" message="Fail: see ./bin/output/compiler_tests_struct_struct_inst.compiler.stderr.log and ./bin/output/compiler_tests_struct_struct_inst.compiler.stdout.log">Fail: see ./bin/output/compiler_tests_struct_struct_inst.compiler.stderr.log and ./bin/output/compiler_tests_struct_struct_inst.compiler.stdout.log</error>
</testcase>
<testcase name="compiler_tests/struct/struct_member_get.c">
<error type="error" message="Fail: see ./bin/output/compiler_tests_struct_struct_member_get.compiler.stderr.log and ./bin/output/compiler_tests_struct_struct_member_get.compiler.stdout.log">Fail: see ./bin/output/compiler_tests_struct_struct_member_get.compiler.stderr.log and ./bin/output/compiler_tests_struct_struct_member_get.compiler.stdout.log</error>
</testcase>
<testcase name="compiler_tests/struct/struct_member_set.c">
<error type="error" message="Fail: see ./bin/output/compiler_tests_struct_struct_member_set.compiler.stderr.log and ./bin/output/compiler_tests_struct_struct_member_set.compiler.stdout.log">Fail: see ./bin/output/compiler_tests_struct_struct_member_set.compiler.stderr.log and ./bin/output/compiler_tests_struct_struct_member_set.compiler.stdout.log</error>
</testcase>
<testcase name="compiler_tests/struct/struct_two_members.c">
<error type="error" message="Fail: see ./bin/output/compiler_tests_struct_struct_two_members.compiler.stderr.log and ./bin/output/compiler_tests_struct_struct_two_members.compiler.stdout.log">Fail: see ./bin/output/compiler_tests_struct_struct_two_members.compiler.stderr.log and ./bin/output/compiler_tests_struct_struct_two_members.compiler.stdout.log</error>
</testcase>
<testcase name="compiler_tests/types/sizeof_char_inst.c">
<error type="error" message="Fail: see ./bin/output/compiler_tests_types_sizeof_char_inst.compiler.stderr.log and ./bin/output/compiler_tests_types_sizeof_char_inst.compiler.stdout.log">Fail: see ./bin/output/compiler_tests_types_sizeof_char_inst.compiler.stderr.log and ./bin/output/compiler_tests_types_sizeof_char_inst.compiler.stdout.log</error>
</testcase>
<testcase name="compiler_tests/types/sizeof_int_inst.c">
<error type="error" message="Fail: simulation did not exit with exit-code 0">Fail: simulation did not exit with exit-code 0</error>
</testcase>
<testcase name="compiler_tests/types/unsigned.c">
<error type="error" message="Fail: see ./bin/output/compiler_tests_types_unsigned.compiler.stderr.log and ./bin/output/compiler_tests_types_unsigned.compiler.stdout.log">Fail: see ./bin/output/compiler_tests_types_unsigned.compiler.stderr.log and ./bin/output/compiler_tests_types_unsigned.compiler.stdout.log</error>
</testcase>
</testsuite>
